cerberus/tests/test_validation.py

```diff
@@ -840,6 +840,7 @@ def test_dependencies_errors():
                 error=('field2', ('field2', 'dependencies'),
                        errors.DEPENDENCIES_FIELD_VALUE,
                        {'field1': ['one', 'two']}, ({'field1': 'three'},)))
+    assert_success({'field1': 'three'}, validator=v)
```

Comment:
I have just modified source code to be compliant with docs.  All test cases are working now, included this one ;)